bin/run-command

```diff
@@ -144,7 +144,7 @@ while true; do
     echo "Timing out task $ecs_task_id waiting for logs"
     exit 1
   fi
-  is_log_stream_created=$(aws logs describe-log-streams --no-cli-pager --log-group-name "$log_group" --query "length(logStreams[?logStreamName==\`$log_stream\`])")
+  is_log_stream_created=$(aws logs describe-log-streams --no-cli-pager --log-group-name "$log_group" --log-stream-name-prefix "$log_stream" | jq -r '.logStreams | length')
   if [ "$is_log_stream_created" == "1" ]; then
```

Comment:
I'm wondering if this if statement down here will trip us up? Can you test that part too pls 🙏🏽

Reply:
Oh duh I didn't read to the end of your PR description sorry 🤦🏽‍♀️

Reply:
It looks to work fine when I ran `run-command` locally. It looped a few times before exiting the loop and printing the logs (the next part of the script).

Even just testing it outside the script, if I run with a valid log group: `aws logs describe-log-streams --no-cli-pager --log-group-name "service/api-dev" --log-stream-name-prefix "api-dev/api-dev/b83f296e29b8412e8446e831150c5518" | jq -r '.logStreams | length'` produces `1`. It I change the log stream name to a random string, it prints `0`